resources/dicts/thoughts/dead/darkforest/exiled.json

```diff
@@ -1,3 +1,63 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "id": "exiled_df_dead_thoughts",
+    "thoughts": [
+      "Is thinking of the dissaproving glares of {PRONOUN/m_c/poss} old Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} train",
+        "Is watching {PRONOUN/m_c/poss} old Clan with scorn hoping for an opportunity to get back at them",
+        "Wants to make {PRONOUN/m_c/poss} old leader pay for exiling {PRONOUN/m_c/object}",
+        "Is planning on making c_n pay for exiling {PRONOUN/m_c/object}"
+        ]
+    },
+    {
+        "id": "exiled_df_evil",
+        "thoughts": [
+            "Is imagining the tree {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} clawing at is an old Clanmate",
+            "Trains extra hard waiting for an opportunity to get {PRONOUN/m_c/poss} revenge",
+            "Knows {PRONOUN/m_c/subject} {VERB/m_c/were/was} right to do what {PRONOUN/m_c/subject} did",
+            "Is cursing {PRONOUN/m_c/poss} old Clan"
+        ],
+    "main_trait_constraint": [
+        "bloodthirsty",
+        "vengeful",
+        "cold",
+        "fierce",
+        "arrogant",
+        "ambitious"
+        ]
+    },
+    {
+        "id": "exiled_df_kind",
+        "thoughts": [
+            "Truly believes {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} misunderstood",
+            "Is sobbing quietly",
+            "Has lost all hope",
+            "Wants nothing more then to fade away",
+            "Is begging a shadowy cat for mercy to no avail",
+            "Is deeply regretting what {PRONOUN/m_c/subject} did to get exiled and end up here",
+            "Is reflecting on what {PRONOUN/m_c/subject} did and why {PRONOUN/m_c/subject} ended up in the Dark Forest"
+        ],
+    "main_trait_constraint": [
+        "compassionate",
+        "faithful",
+        "loving",
+        "patient",
+        "thoughtful",
+        "Insecure",
```

Comment:
insecure is actually closer to a mean trait; its aggressiveness requirement is high and it tends to lead cats with the trait to being more hostile or violent. maybe replace it with the nervous trait here?

Reply:
good point, will swap that out now